### DISCUSSION TITLE
Change wording of the "standalone" option in the create-nx-workspace

### DISCUSSION THREAD

**Discussion by moniuch** [nrwl/nx#23407] [General]
- [ ] I'd be willing to implement this feature ([contributing guide](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md))

## Motivation

I find the `create-nx-workspace` command a bit counter-intuitive when it comes to naming and availability of some options.
My goal was to create an angular workspace holding multiple apps.
At first attempt I misunderstood the word "standalone" as the new NgModule-less way of building angular apps. Then I found out that this actually translates to a "single-app" workspace. 

## Suggested Implementation

So maybe something alongside the lines of _single app_, _multi app_ would be a better wording choice to describe the command output.

**Comment by FrozenPandaz** [comment]
Hopefully as the Angular Standalone mode becomes more default, this won't be as confusing?

cc @leosvelperez @Coly010
